Cargo.toml

```diff
@@ -1,15 +1,16 @@
 [package]
 name = "vtebench"
-version = "0.1.2"
+version = "0.2.0"
+authors = ["Christian Duerr <contact@christianduerr.com>", "Joe Wilm <joe@jwilm.com>"]
 license = "Apache-2.0"
-authors = ["Joe Wilm <joe@jwilm.com>"]
 description = "Benchmark terminal emulators"
-documentation = "https://docs.rs/vtebench"
-repository = "https://github.com/alacritty/vtebench"
+readme = "README.md"
+homepage = "https://github.com/alacritty/vtebench"
+edition = "2018"
 
 [dependencies]
-structopt-derive = "0.2.10"
-structopt = "0.2.10"
-terminfo = "0.6.1"
-failure = "0.1.2"
-rand = "0.5.5"
+structopt-derive = "0.4"
+structopt = "0.3"
+terminfo = "0.7"
+failure = "0.1"
+rand = "0.7"
```

Comment:
Can we go ahead and add the patch version to all of these?

Reply:
I'm not entirely sure that we should, since there's no real reason for it, and users may have a version behind on their systems, though,

Reply:
Well did you test it with the 0.X.0 versions? If not, we definitely should. Otherwise we should test with the versions set to 0.X.0 and then specify those exactly, to make it clear that's what works. There's rarely ever a good reason to omit the patch version.